contrib/datawave-quickstart/bin/services/hadoop/bootstrap.sh

```diff
@@ -129,6 +129,7 @@ function hadoopStatus() {
         local _pid
         local _opt=pid
 
+        DW_HADOOP_PID_LIST="$(eval "${DW_HADOOP_CMD_FIND_ALL_PIDS}")"
```

Comment:
Sorry....last issue, I think:

I noticed that this function still requires ```hadoopIsRunning```  (line 128) to return success in order to begin evaluating the status of each process in the loop. So, even if there are some processes running, the status printed out will be... 
```
[DW_WARN] - X is not running
```
...for all of them b/c the pid count is less than 5

So, this line just needs to be moved up with ```test -n ...``` added, exactly like your fix for ```accumuloStatus``` above.
```
...
L128      DW_HADOOP_PID_LIST="$(eval "${DW_HADOOP_CMD_FIND_ALL_PIDS}")"
L129      test -n "${DW_HADOOP_PID_LIST}" && {
```